ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,287 @@
+SITE_HOST: 'enikshay.in'
+#CAS_SITE_HOST: 'cas.commcarehq.org'
+#ENIKSHAY_SITE_HOST: 'enikshay.commcarehq.org'
```

Comment:
phones are currently submitting to https://enikshay.commcarehq.org. Everything @emord said is true about needing this to be around after we migrate to Ctrl-S.